src/js/program/background-media.ts

```diff
@@ -12,34 +18,36 @@
  * @property {Promise} nextMedia - A promise representing the media loading in the background. Resolves with the mediaController.
  * @constructor
  */
-export default function BackgroundMedia() {
-    let currentMedia = null;
-    let nextMedia = null;
+export default function BackgroundMedia(): object {
+    let currentMedia: MediaObject | null = null;
```

Comment:
Based on the JSDOC it looks like this might actually be a `MediaController` type? 
Though those JSDOC comments are confusing me a little.

Reply:
That's correct. See `background.currentMedia` in program-controller. `BackgroundMedia` is only used in program-controller.